feature-libs/product-configurator/rulebased/components/product-title/configurator-product-title.component.spec.ts

```diff
@@ -73,7 +72,22 @@ const orderEntryconfig: Configurator.Configuration = {
   },
 };
 
-const orderEntryconfigWoOverview: Configurator.Configuration = {
+const cartEntryConfig: Configurator.Configuration = {
+  ...ConfiguratorTestUtils.createConfiguration(
+    CONFIG_ID,
+    ConfiguratorModelUtils.createOwner(
+      CommonConfigurator.OwnerType.CART_ENTRY,
+      '0'
+    )
+  ),
+  productCode: PRODUCT_CODE,
+  overview: {
+    configId: CONFIG_ID,
+    productCode: PRODUCT_CODE,
```

Comment:
For the cart entry we expect the product to be used from config and not from the overview. Providing the same product code in both locations within the test data does somehow invalidate your test.

Reply:
Please, check again if now is okay because I refactored a lot.